sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitor.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.opentelemetry.exporter;
+
+/**
+ * A convenient class to instantiate an instance of {@link AzureMonitorInstaller}
+ */
+public final class AzureMonitor {
+
+    /**
+     * Construct an intance of {@link AzureMonitorInstaller}
+     * @return a new {@link AzureMonitorInstaller} object
+     */
+    public static AzureMonitorInstaller installer() {
```

Comment:
A private constructor could be added ton force the use of the static method.

Reply:
```suggestion
    public static AzureMonitorInstaller initialize() {
```

Reply:
and then keep the install(sdkBuilder) in AzureMonitorInstaller. Installer.install() makes sense to me.

Reply:
> A private constructor could be added ton force the use of the static method.

private constructor is at line 19.